roles/build_openstack_packages/tasks/create_repo.yml

```diff
@@ -51,6 +51,9 @@
       priority=1
     dest: "{{ cifmw_bop_gating_repo_dest }}/gating.repo"
 
+- name: Serve gating repo
+  ansible.builtin.import_tasks: serve_gating_repo.yml
+
```

Comment:
for future maintenance we could consider adding vars: to this import so we can add 8766 as param avoiding repetition there in serve_gating_repo.yaml
```
- name: Serve gating repo
  ansible.builtin.import_tasks: serve_gating_repo.yml
  vars:
      gating_port: 8766
```
```

Reply:
Thank you @marios for the suggestion!. Sounds good.